### PR TITLE
Feat/upgrade electron

### PR DESCRIPTION
**Pull request by KeKo6988**
Upgrade Electron to 12.0.2, which now supports Apple M1! (as of [Electron 11](https://www.electronjs.org/blog/electron-11-0)).

Brings with it changes to support Corda, which will be reverted in `revert/corda` once this makes it's way into `develop`

### CODE REVIEW

File: src/main/index.js [hunk 1]
```diff
@@ -219,7 +219,8 @@ app.on('ready', async () => {
     icon: getIconPath(),
     webPreferences: {
       nodeIntegration: true,
-      enableRemoteModule:true
+      contextIsolation: false,
+      enableRemoteModule: true
```
Comment (jeffsmale90) [inline src/main/index.js:223]: `In Electron 10, the remote module is now disabled by default. `

File: src/main/index.js [hunk 2]
```diff
@@ -219,7 +219,8 @@ app.on('ready', async () => {
     icon: getIconPath(),
     webPreferences: {
       nodeIntegration: true,
-      enableRemoteModule:true
+      contextIsolation: false,
```
Comment (jeffsmale90) [inline src/main/index.js:222]: `In Electron 12, contextIsolation will be enabled by default.`

File: package.json [hunk 3]
```diff
@@ -67,7 +67,7 @@
     "node-fetch": "2.6.0",
     "node-localstorage": "^2.1.6",
     "node-ssh": "7.0.0",
-    "pg": "7.14.0",
+    "pg": "8.6.0",
```
Comment (jeffsmale90) [inline package.json:70]: This is to support Corda, I'll undo this change in `revert/corda` branch

File: src/renderer/screens/loader/LoaderScreen.scss [hunk 4]
```diff
@@ -57,7 +57,7 @@
 
   .LoaderProgress {
     position: absolute;
-    margin-top: 7em;
+    margin-top: 15em;
```
Comment (jeffsmale90) [inline src/renderer/screens/loader/LoaderScreen.scss:60]: I _think_ this is to support Corda, I'll look into this in `revert/corda` branch

**Review by davidmurdoch — APPROVED** [review]

**Pull request merged**